code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -70,28 +80,43 @@
 	icon_state = "nether"
 	max_integrity = 50
 	spawn_time = 600 //1 minute
-	max_mobs = 15
+	max_mobs = 10
 	icon = 'icons/mob/nest.dmi'
 	spawn_text = "crawls through"
 	mob_types = list(/mob/living/simple_animal/hostile/netherworld/migo, /mob/living/simple_animal/hostile/netherworld, /mob/living/simple_animal/hostile/netherworld/blankbody)
-	faction = list("nether")
+	faction = list("nether", "mining", "cult")
+	var/blankckey = null
 
 /obj/structure/spawner/nether/Initialize(mapload)
 	.=..()
 	START_PROCESSING(SSprocessing, src)
 
 /obj/structure/spawner/nether/examine(mob/user)
 	..()
-	if(isskeleton(user) || iszombie(user))
-		to_chat(user, "A direct link to another dimension full of creatures very happy to see you. <span class='nicegreen'>You can see your house from here!</span>")
+	if(iscultist(user))
+		to_chat(user, "A direct link to another dimension full of creatures very happy to see your God again. <span class='notice'>They are patiently waiting to be re-enlisted.</span>") //By touching the portal on harm or disarm intent, you will recall all the monsters spawned by the portal. By touching it on help or grab intent, you will be able to move it around.</span>")
 	else
 		to_chat(user, "A direct link to another dimension full of creatures not very happy to see you. <span class='warning'>Entering the link would be a very bad idea.</span>")
 
 /obj/structure/spawner/nether/attack_hand(mob/user)
 	. = ..()
-	if(isskeleton(user) || iszombie(user))
```

Comment:
Again, why remove this?

Reply:
I changed it to give a special message to cultists rather than zombies or skeletons.

The idea was that cultists could do more with the portal, which is what the commented out features are, however if they are completely removed later on (which they probably will) I'll revert it back to just giving a special message for skeletons and zombies.